pkg/template/registry/rest.go

```diff
@@ -49,7 +49,13 @@ func (s *REST) Create(ctx kapi.Context, obj runtime.Object) (<-chan apiserver.RE
 			glog.V(1).Infof(utilerr.NewAggregate(err).Error())
 		}
 
-		if err := template.AddConfigLabels(cfg, labels.Set{"template": tpl.Name}); len(err) > 0 {
+		objectLabels := labels.Set{"template": tpl.Name}
```

Comment:
We need to stop doing this now that we have the other field.  I don't think we should add a label by default anymore.  The process command can support a label selector and that's enough.